apps/typegpu-docs/astro.config.mjs

```diff
@@ -103,12 +112,14 @@ export default defineConfig({
           ],
         },
         {
-          label: '🙋 FAQ',
+          label: 'FAQ',
           slug: 'faq',
         },
       ]),
     }),
-    tailwind(),
+    tailwind({
+      applyBaseStyles: false,
```

Comment:
that's because instead we import tailwind.css file? what's the difference?

Reply:
Yes. That's what the Starlight docs about Tailwind integration now say. I assume its because Starlight applies its own reset styles.